services/vaadin-directory/vaadin-directory-rating-count.service.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { metric } = require('../text-formatters')
+const { floorCount: floorCountColor } = require('../color-formatters')
+const { BaseVaadinDirectoryService } = require('./vaadin-directory-base')
+
+module.exports = class VaadinDirectoryRatingCount extends BaseVaadinDirectoryService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'vaadin-directory',
+      pattern: ':which(rc|rating-count)/:packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Vaadin Directory',
+        pattern: 'rating-count/:packageName',
+        namedParams: { packageName: 'vaadinvaadin-grid' },
+        staticPreview: this.render({ ratingCount: 6 }),
+        keywords: ['vaadin-directory', 'rating-count', 'rating count'],
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'rating count' }
+  }
+
+  static render({ ratingCount }) {
+    return {
+      message: `${metric(ratingCount)} total`,
+      color: floorCountColor(ratingCount, 5, 50, 500),
+    }
+  }
+
+  async handle({ which, packageName }) {
```

Comment:
Looks like we aren't using `which` here, so I take it the `rc` and `rating-count` there are for BC?

Reply:
Yes exactly. There are several vaadin badges which have more than one alias for the same badge:

* `rc|rating-count`
* `star|stars`
* `rd|release-date`
* `v|version`

all do the same thing.

Maybe it would be better to call the param `:alias` instead of `:which`?
I'm not massively keen to do it with a redirect class in this case as it requires quite a bit more boilerplate to do it that way.

Reply:
My initial thought here at first glance was that eslint would bark at `which` being unused but looks like I was wrong there. 

Given that putting the aliases in the route param doesn't prevent using a `pattern` (as articulated in #3329), I don't have any objections with this over redirectors. 👍